### ISSUE TITLE
No transition when toggling a collapsed Navbar in 4.5 anymore

### ISSUE THREAD

**Issue by new-mikha** [twbs/bootstrap#31237]
- OS: Windows 10, version 1809
- Browser: Chrome 83.0.4103.116 (Official Build) (64-bit)

Navbar 4.0 have an animation when the toggle button pressed, but 4.1.3 doesn't have it anymore.

The official manual page have this bug, so no need in separate example.

For both links below do the following:
- Shrink the page width (maybe turn the DevTools on, and then toggle device layout, or just resize the browser)
- Navigate down to the nearest example on the page
- Press the toggle button: the 4.0 page does have the transition - the toggling of the collapsed navbar is animated, while 4.1 does NOT have it.

https://getbootstrap.com/docs/4.0/components/navbar/#how-it-works
https://getbootstrap.com/docs/4.5/components/navbar/#how-it-works

The same is happening in isolated tests too, when the navbar HTML snippets are copied from those pages to "hello world" templates with corresponding versions.

Looks like a serious bug??

Below are the screen recordings. First is 4.0, second is 4.5, I hope you see the difference.

![Navbar · Bootstrap - 4 0](https://user-images.githubusercontent.com/5769929/86506835-906f6980-be16-11ea-9f9b-e38b7cd78e7f.gif)  ![Navbar · Bootstrap v4 5 (1)](https://user-images.githubusercontent.com/5769929/86507069-864e6a80-be18-11ea-932d-4d4bd69fea21.gif)

Edit: I noticed that on real Android it's working correctly. However on the desktop it's still a bug.

**Comment by patrickhlauke** [comment]
see https://getbootstrap.com/docs/4.1/getting-started/accessibility/#reduced-motion (and for more detail if needed https://developer.paciellogroup.com/blog/2019/05/short-note-on-prefers-reduced-motion-and-puzzled-windows-users/)

**Comment by new-mikha** [comment]
Wow, thank you very much @patrickhlauke ! I surely didn't know about that.
To my excuse, I should say that I have had a thought "oh ok maybe there is some kind of a switch now", and did a search in the getbootstrap.com first. Neither of the searches that I tried has brought this little section of the doco up. I tried "transition" and "animation" - I did not think about "motion" though. Maybe if there are really plenty of false bug reports like this one, the section needs to be made more visible. Thanks for the explanation!
